cscs-checks/apps/espresso/espresso_check.py

```diff
@@ -49,10 +49,10 @@ def __init__(self, **kwargs):
 
         self.reference = {
             'dom:mc': {
-                'sec': (217.0, None, 0.15),
+                'sec': (158.8, None, 0.05),
```

Comment:
I think that `158.8` is the same as `159.0`, so I would leave it as `159.0`. What do you think?